activiti-image-generator/pom.xml

```diff
@@ -0,0 +1,145 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>Activiti - Image Generator</name>
+  <artifactId>activiti-image-generator</artifactId>
+  <packaging>jar</packaging>
+
+  <parent>
+    <groupId>org.activiti</groupId>
+    <artifactId>activiti-root</artifactId>
+    <version>7.0.0-SNAPSHOT</version>
+  </parent>
+
+  <properties>
```

Comment:
We need to remove all this extra configuration. Look at the other modules, let's not add all the osgi exports and profiles. Remove the plugin and build section as well. Just keep the dependencies.